netlify.toml

```diff
@@ -3,6 +3,7 @@
 [build]
   # Default build command.
   command = "yarn build"
+  functions = "src/lambda"
```

Comment:
@will0101 This is most likely the issue. We've told Netlify to look in the `src/lambda` directory for functions, but that directory does not exist. They're in `src/functions`

Reply:
@tylergaw The `src/lambda` directory has been added to the `.gitignore` file. This directory is for serving the code we work on in the `src/functions` folder.

Reply:
Fixed!